### PR TITLE
Update dependency react-instantsearch-dom to ^6.26.0

### PR DESCRIPTION
**Pull request by mustafa-kapadia1483**
Fixes to Issue #759

### CODE REVIEW

**Comment by w3bdesign** [comment]
@mustafa-kapadia1483 

Getting the following build error:

`Error: Specified images.domains should be an Array of strings received invalid values ().`

File: components/AlgoliaSearch/SearchResults.component.jsx [hunk 1]
```diff
@@ -35,12 +36,15 @@ const SearchResults = ({
         as={`/produkt/${trimmedProductName}?id=${objectID}`}
         passHref
       >
-        <div className="flex p-6 bg-white">
+        <a className="flex p-6 bg-white">
```
Comment (w3bdesign) [inline components/AlgoliaSearch/SearchResults.component.jsx:39]: @mustafa-kapadia1483 

I don't think this is needed.

File: components/AlgoliaSearch/SearchResults.component.jsx [hunk 2]
```diff
@@ -62,7 +66,7 @@ const SearchResults = ({
               {trimmedStringToLength(short_description, 30)}
             </span>
           </div>
-        </div>
+        </a>
```
Comment (w3bdesign) [inline components/AlgoliaSearch/SearchResults.component.jsx:69]: I don't think this is needed either.

File: components/Cart/CartPage/CartItemsContainer.component.jsx [hunk 3]
```diff
@@ -97,9 +97,9 @@ const CartItemsContainer = () => {
             />
             <div className="mt-4">
               <Link href="/kasse" passHref>
-                <button className="px-4 py-2 font-bold bg-white border border-gray-400 border-solid rounded hover:bg-gray-400">
+                <a className="px-4 py-2 font-bold bg-white border border-gray-400 border-solid rounded hover:bg-gray-400">
```
Comment (w3bdesign) [inline components/Cart/CartPage/CartItemsContainer.component.jsx:100]: Why is this no longer a button?

File: components/Product/IndexProducts.component.jsx [hunk 4]
```diff
@@ -47,23 +52,16 @@ const IndexProducts = ({ products }) => {
...
-                          id="product-image"
-                          className="transition duration-500 ease-in-out transform cursor-pointer hover:grow hover:shadow-lg hover:scale-105"
-                          alt={name}
-                          src={image.sourceUrl}
-                        />
-                      ) : (
-                        <img
-                          id="product-image"
-                          className="transition duration-500 ease-in-out transform cursor-pointer hover:grow hover:shadow-lg hover:scale-105"
+                      <div className="h-80 relative cursor-pointer">
```
Comment (w3bdesign) [inline components/Product/IndexProducts.component.jsx:55]: What happened to the check for image here?

**Comment by mustafa-kapadia1483** [comment]
Did you create and new env variable WP_DOMAIN ? It should be the domain name of the WordPress server, so that it can be passed to next.config.js

**Review by w3bdesign — CHANGES_REQUESTED** [review]
See comments above

@mustafa-kapadia1483

File: components/Product/SingleProduct.component.jsx [hunk 5]
```diff
@@ -51,22 +55,15 @@ const SingleProduct = ({ product }) => {
       ) : (
         <div className="container flex flex-wrap items-center pt-4 pb-12 mx-auto ">
           <div className="grid grid-cols-1 gap-4 mt-16 lg:grid-cols-2 xl:grid-cols-2 md:grid-cols-2 sm:grid-cols-2">
-            {image && (
-              <img
+            <div className="h-auto p-8 transition duration-500 ease-in-out transform xl:p-2 md:p-2 lg:p-2 hover:grow hover:shadow-lg hover:scale-105">
```
[the remark this answers is not in the document]
Comment (mustafa-kapadia1483) [reply components/Product/SingleProduct.component.jsx:58]: I extracted it above aa it was showing as duplicate code. What I done is:
`let sourceUrl = image sourceUrl || NEXT_PUBLIC_PLACEHOLDER_LARGE_IMAGE_URL`

**Comment by w3bdesign** [comment]
> Did you create a new env variable `WP_DOMAIN` ? It should be the domain name of the WordPress server, so that it can be passed to next.config.js

I added WP_DOMAIN to Environment Variables in Vercel now, can you try pushing a new deploy?

File: components/Cart/CartPage/CartItemsContainer.component.jsx (hunk 3 above)
Comment (mustafa-kapadia1483) [reply components/Cart/CartPage/CartItemsContainer.component.jsx:100]: passHref does not pass onto native HTML elements other than a tag.

**Review by w3bdesign — APPROVED** [review]

**Comment by mustafa-kapadia1483** [comment]
Sure, will update the same

**Comment by w3bdesign** [comment]
@mustafa-kapadia1483 

I did a quick visual inspection and can see some changes, can you doublecheck all the changes you did with my existing live deployment?

See screenshots below.

bilde

bilde

**Comment by w3bdesign** [comment]
I had some issues with Next/image which was why I have delayed implementing it. 

For example, it seems to add some custom CSS of it own. 
It may be possible that it is better avoiding it for now.

**Comment by w3bdesign** [comment]
> Maybe you can keep it for now if perfecting design is not a major concern right now. Otherwise it's better to avoid it for atleast dynamic sized images sections.

It's one of my main portfolio projects, so having a perfect design is important on this particular project. 

You can revert the Next/image changes if it is easier.

**Pull request closed without merging**